The original patch only covers meson build. This file adds automake support.

Ref: https://github.com/NixOS/nix/pull/12115

diff --git a/configure.ac b/configure.ac
index 985815990..5c0ffce5e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -386,6 +386,7 @@ AS_CASE(["$enable_markdown"],
       CXXFLAGS="$LOWDOWN_CFLAGS $CXXFLAGS"
       have_lowdown=1
       AC_DEFINE(HAVE_LOWDOWN, 1, [Whether lowdown is available and should be used for Markdown rendering.])
+      AC_DEFINE(HAVE_LOWDOWN_1_4, 1, [Whether lowdown version 1.4.0 is available.])
     ], [
       AS_IF([test "x$enable_markdown" == "xyes"], [AC_MSG_ERROR([--enable-markdown was specified, but lowdown was not found.])])
     ])
